src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,25 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    private static final int ADULT = 18;
+    private static final int MIN_LENGTH_PASSWORD = 6;
+    private static final StorageDao STORAGEDAO = new StorageDaoImpl();
```

Comment:
```suggestion
    private final StorageDao storageDao = new StorageDaoImpl();
```
No need to make class objects constants, only final